npm/test-integration.js

```diff
@@ -61,8 +61,7 @@ module.exports = function (exit) {
                     collection: test.collectionJSON,
                     environment: test.environmentJSON,
                     globals: test.globalsJSON,
-                    iterationData: test.dataCSV || test.dataJSON,
-                    abortOnError: true
```

Comment:
We need to have a hidden option here, that we can pass to the library. With stopOnFailure, we won't see any errors in the callback, because it's a graceful stop.